src/components/controls/keyboard-controls.js

```diff
@@ -0,0 +1,94 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
It's interesting that I think a lot of people say that the Web was meant to work everywhere. I think for WebVR, responsiveness means that it works on multiple HMDs and platforms, not that it works across different mediums.